packages/sdk-codegen/package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "author": "Looker",
   "license": "MIT",
-  "private": false,
+  "private": true,
```

Comment:
Interesting. Didn't know sdk-codegen is not meant to be published on npm

Reply:
It will be after we fix the package versioning for CD. It shouldn't follow looker release versioning like the SDKs do